scripts/sanitycheck

```diff
@@ -2341,15 +2341,28 @@ def main():
                     COLOR_YELLOW,
                     COLOR_NORMAL,
                     reason))
+    from copy import deepcopy
 
-    ts.instances = OrderedDict(
+    tmp = OrderedDict(
         sorted(ts.instances.items(), key=lambda t: t[0]))
 
+    ts.instances = deepcopy(tmp)
+
     if options.save_tests:
         ts.run_report(options.save_tests)
         return
 
     if options.subset:
+        # This will put all native_posix first and will result in all code
+        # coverage reports coming from once CI job, avoiding the need for
+        # merging coverage reports.
+
+        for item in tmp.items():
+            if item[0].startswith('native_posix'):
+                ts.instances.move_to_end(item[0], False)
+            else:
+                ts.instances.move_to_end(item[0], True)
```

Comment:
... but then what does this line does? Is it really needed?

Reply:
yes, thats not needed anymore, i was trying something else before i ended up doing this.